Submitted By: DJ Lucas (dj AT linuxfromscratch DOT org)
Date: 2005-10-06
Initial Package Version: 1.5.0
Origin: JDK-6.0 (Mustang) snapshot
Upstream Status: Fixed in 6.0
Description: Fixes syntax for use with nptl.

$LastChangedBy$
$Date$


diff -Naur jdk-1.5.0_04-orig/deploy/src/javaws/solaris/native/system_md.c jdk-1.5.0_04/deploy/src/javaws/solaris/native/system_md.c
--- jdk-1.5.0_04-orig/deploy/src/javaws/solaris/native/system_md.c      2005-10-02 17:48:50.000000000 -0500
+++ jdk-1.5.0_04/deploy/src/javaws/solaris/native/system_md.c   2005-10-02 17:57:00.000000000 -0500
@@ -885,3 +885,8 @@

   return strcat(temp, dbg_suffix);
 }
+
+char *sysGetNetProperty() {
+  return NULL;
+}
+
